src/evict/evict_lru.c

```diff
@@ -1885,14 +1885,20 @@ __evict_walk_tree(WT_SESSION_IMPL *session, WT_EVICT_QUEUE *queue, u_int max_ent
      */
     switch (btree->evict_start_type) {
     case WT_EVICT_WALK_NEXT:
+        if (btree->evict_ref == NULL && (++btree->linear_walk_restarts) & 1)
```

Comment:
What is & 1 for?

Reply:
Each time this is executed, the behavior alternates between regular linear walk and random one.

Reply:
Can we wrap this in a macro or add a comment?

Reply:
Added a comment.